frontend/src/app/home/home.component.html

```diff
@@ -0,0 +1 @@
+<div class="center-message">Welcome to Dog-Image-Classifier</div>
```

Comment:
I like where your head is at. We can use this component in the future to give a quick rundown of how to use the app.